### ISSUE TITLE
Can you provide the setting that reaches to 85.x% accuracy?

### ISSUE THREAD

**Issue by GodOfSmallThings** [ZJULearning/pixel_link#48]
@dengdan 
I tried to reproduce the result that you've reached, which is 85%. However, I failed to make that result. It goes as much as 83% in my experiments. Can you provide the setting(e.g hyper parameters that you've used for pretraining and finetuning) using both Synth Text and ICDAR datasets? 

Thank you.

**Comment by mn7142** [comment]
@GodOfSmallThings @kulkarnivishal 
please help me to evaluate my result
how can you calculate your result?

**Comment by CXY573** [comment]
how can you calculate your result?

**Comment by XiaoYigwr** [comment]
@GodOfSmallThings @kulkarnivishal @mn7142 
How did you test the results of the model?你们是阿如何检验结果的
Could you share the script?可以分享一下脚本吗
I don't konw how to do!

**Comment by SungAnGweon** [comment]
@XiaoYigwr  
Go to http://rrc.cvc.uab.es/?ch=4&com=mymethods&task=1 after login.
There is standalone evaluation script and it is easy to use.

**Comment by GodOfSmallThings** [comment]
@XiaoYigwr Go to http://rrc.cvc.uab.es/?ch=4&com=mymethods&task=1 after login. There is standalone evaluation script and it is easy to use.

**Comment by PeterStone007** [comment]
standalone evaluation script is not available.  do you have other source?
